### PR TITLE
updated package.json to use SDK-23.1.1

### PR DESCRIPTION
**Pull request by niallriddell**
Tested with latest mediaco on lab server with 23.1.1

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,7 +37,7 @@
     "@lion/radio-group": "^0.21.1",
     "@lion/select": "^0.17.1",
     "@lion/textarea": "^0.16.1",
-    "@pega/constellationjs": "SDK-8.8.2",
+    "@pega/constellationjs": "SDK-23.1.1",
```
Comment (tumms2021389) [inline package.json:40]: Use ~23.1.1 version instead of SDK tag

**Comment by niallriddell** [comment]
Seems contrary to advice on npm.

**Comment by timjmartel** [comment]
The version doesn't match SDK versioning rules.  Should be 23.1.10.

File: package.json (hunk 1 above)
Comment (timjmartel) [reply package.json:40]: Doesn't match SDK versioning rules.  Last digit starts at 10 to avoid confusion with infinity patch version.

**Comment by tumms2021389** [comment]
> The version doesn't match SDK versioning rules. Should be 23.1.10.

Constellationjs files npm versions are matching with Infinity version not SDK version so it will be 23.1.1

**Pull request merged**